backend/prisma/migrations/20240627173035_workout_info/migration.sql

```diff
@@ -0,0 +1,24 @@
+-- CreateTable
+CREATE TABLE "Exercise" (
+    "id" SERIAL NOT NULL,
+    "workoutId" INTEGER NOT NULL,
+    "name" TEXT NOT NULL,
+
+    CONSTRAINT "Exercise_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Set" (
+    "id" SERIAL NOT NULL,
+    "exerciseId" INTEGER NOT NULL,
+    "weight" INTEGER NOT NULL,
```

Comment:
nit: do we need DECIMAL for weight?

Reply:
Could be useful for small weights

Reply:
Also I believe it was a typo, reps should be Int and weight should be a float. Fixed! Nice catch.